manifests/puppet_ruby.pp

```diff
@@ -2,7 +2,56 @@
 #
 # A description of what this class does
 #
+# @api private
+#
 # @example
 #   include hdm::puppet_ruby
 class hdm::puppet_ruby {
+  package { 'bundler':
+    ensure   => present,
+    provider => 'puppet_gem',
+  }
+  group { $hdm::group:
+    ensure => present,
+  }
+  user { $hdm::user:
+    ensure => present,
+    gid    => $hdm::group,
+    shell  => '/sbin/nologin',
+  }
+  vcsrepo { $hdm::hdm_path:
+    ensure   => present,
+    provider => 'git',
+    source   => $hdm::git_url,
+    revision => $hdm::version,
+    user     => $hdm::user,
+    group    => $hdm::group,
+  }
+  exec { 'bundle install':
```

Comment:
sollte das exec auch mit dem neuen user/gruppe aufgerufen werden?

Reply:
The code must be owned by root user. HDM is running as hdm user, so hdm app can not modify any files.